src/components/ble/NimbleController.cpp

```diff
@@ -376,6 +380,22 @@ void NimbleController::NotifyBatteryLevel(uint8_t level) {
   }
 }
 
+void NimbleController::SwitchAirplaneMode(bool enabled) {
+  if (enabled) {
+    if (bleController.IsConnected()) {
+      bleController.SetConnectState(Ble::ConnectStates::Airplane);
+      ble_gap_terminate(connectionHandle, BLE_ERR_REM_USER_CONN_TERM);
+    } else {
+      bleController.SetConnectState(Ble::ConnectStates::Airplane);
+      ble_gap_adv_stop();
+    }
+  } else {
+    bleController.SetConnectState(Ble::ConnectStates::Disconnected);
+    fastAdvCount = 0;
+    StartAdvertising();
+  }
+}
+
```

Comment:
I would rather have this be two functions without a boolean parameter.

Reply:
I disagree. I see no reason to make the code larger.